tests/sentry/models/test_group.py

```diff
@@ -16,6 +14,13 @@
 
 
 class GroupTest(TestCase):
+    def setUp(self):
+        super(GroupTest, self).setUp()
+        self.min_ago = (timezone.now() - timedelta(minutes=1)).isoformat()[:19]
+        self.two_min_ago = (timezone.now() - timedelta(minutes=2)).isoformat()[:19]
+        self.sec_ago = (timezone.now() - timedelta(seconds=1)).isoformat()[:19]
+        self.two_sec_ago = (timezone.now() - timedelta(seconds=1)).isoformat()[:19]
```

Comment:
Unless you are moving close to the speed of light, thus introducing relativistic effects like time dilation, this is still one sec ago and not two sec ago.
Technically we do not know where travis runs CI so we cannot exclude the relativistic option.